compiler/src/test/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/ContributesBindingProcessorTest.kt

```diff
@@ -131,13 +164,19 @@ class ContributesBindingProcessorTest {
         }
     }
 
-    @Test
-    fun `it's an error when there are multiple super types`() {
+    @ParameterizedTest(name = "annotation: {0}")
```

Comment:
Instead of parameterizing so many tests, I'd like to have a few separate ones for the new annotations. Parameterized tests are much harder to read and from past experience slight variations quickly sneak in.

Reply:
Fair. I saw a few parameterized tests already in the codebase, so I though that was the preferred route here. As we'll be going with the separate module route this won't be an issue anyway.